### PR TITLE
[43680] Move existing options in notification settings

### PR DESCRIPTION
**Pull request by bsatarnejad**
Change some titles and descriptions in notification setting page and Notification setting table.

https://community.openproject.org/work_packages/43680/activity

### CODE REVIEW

**Review by dombesz — APPROVED** [review]
LGTM! 👍
I added a commit to change the `@mentioned` to `Mentioned` because it looked a bit inconsistent and redundant in the notifications sub menu. Once the @marcalcobe approves it, I will merge it.

image
TO ->
image

image
TO ->
image

**Comment by marcalcobe** [comment]
Sorry for not responding here @bsatarnejad and @dombesz.
After discussing with Niels no problem at all so we can proceed with this topic and merge it.

**Comment by lindenthal** [comment]
Hi @dombesz,
Thanks for your remark concerning the @metion inconsistency. I think we need to harmonize the translation strings. We are using too many different ones. 
We created a new work package here: https://community.openproject.org/projects/14/work_packages/44541/activity
Please feel free to address this in this PR or in a separate one. Your call.

Best
Niels

**Pull request merged**
